wn/web.py

```diff
@@ -208,6 +209,19 @@ def make_synset(ss: wn.Synset, request: Request, basic: bool = False) -> dict:
     return d
 
 
+# Exception handlers
+
+async def http_exception_handler(request: Request, exc: Exception):
+    status_code = exc.status_code if hasattr(exc, 'status_code') else 500
+    return JSONResponse({
+        "error": {
+            "status": status_code,
+            "message": exc.detail if hasattr(exc, 'detail') else str(exc),
```

Comment:
This could also be a `getattr` call. There's a case for leaving it as an inline conditional if `str(exc)` is something you don't want to call unless it's needed, but I don't think that's a big concern here.